modules/microshift-ingress-controller-config.adoc

```diff
@@ -91,7 +116,105 @@ ingress:
+
+|`httpCaptureCookies`
+|Specifies HTTP cookies that you want to capture in access logs. If the `httpCaptureCookies` field is empty, access logs do not capture the cookies. Default value is empty. Configuring `ingress.accessLogging.httpCaptureCookies` automatically enables ingress access logging. For any cookie that you want to capture, you must also set the `matchType` and `maxLength` parameters.
+
+* For example:
++
+[source,yam]
+----
+  httpCaptureCookies:
+  - matchType: Exact
+    maxLength: 128
+    name: MYCOOKIE
+----
+
+|`httpCaptureCookies.matchType`
+|Specifies whether the field name of the cookie exactly matches the capture cookie setting or is a prefix of the capture cookie setting. Valid values are `Exact` for an exact string match and `Prefix` for a string prefix match.
+
+* If the `Exact` setting is used, a name must also be specified in the `httpCaptureCookiesname` field.
+* If the `Prefix` setting is used, a prefix must be specified in the `httpCaptureCookiesnamePrefix` field. For example, specifying matchType "Prefix" and namePrefix "foo" will capture a cookie named "foo" or "foobar" but not one named "bar". The first matching cookie is captured.
+
+|`httpCaptureCookies.maxLength`
+|Specifies tha maximum length of the cookie that is logged, which includes the cookie name, cookie value, and
+one-character delimiter. If the log entry exceeds this length, the value is truncated in the log message. The ingress controller might impose a separate bound on the total length of HTTP headers in a request. The minimum value is `1` byte, maximum value is `1024` bytes.
+
+|`httpCaptureCookies.name`
+|Specifies the exact name used for a cookie name match as set in the `httpCaptureCookies.matchType` parameter. The value must be a valid HTTP cookie name as defined in link:https://datatracker.ietf.org/doc/html/rfc6265#section-4.1[RFC 6265 section 4.1]. The minimum length is `0` bytes and the maximum length is `1024` bytes.
+
+|`httpCaptureCookies.namePrefix`
+|Specifies the prefix for a cookie name match as set in the `httpCaptureCookies.matchType` parameter. The value must be a valid HTTP cookie name as defined in link:https://datatracker.ietf.org/doc/html/rfc6265#section-4.1[RFC 6265 section 4.1]. The minimum length is `0` bytes and the maximum length is `1024` bytes.
+
+|`httpCaptureHeaders`
+|Defines the HTTP headers that should be captured in the access logs. This field is a list and allows capturing request and response headers independently. If this field is empty, no headers are captured. This option only applies to 1) plain text HTTP connections and 2) to secure HTTP connections for which the ingress controller terminates encryption: for example, edge-terminated or reencrypt connections. Headers cannot be captured for TLS `passthrough` connections. Configuring `ingress.accessLogging.httpCaptureCookies` automatically enables ingress access logging.
```

Comment:
here, `ingress.accessLogging.httpCaptureCookies` should be `ingress.accessLogging.httpCaptureHeaders`